eng/Version.Details.xml

```diff
@@ -80,9 +76,9 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>03497f16b93264158c28cee4eea767b53bc750f5</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.NETCore.Platforms" Version="3.0.0-preview.19105.1">
+    <Dependency Name="Microsoft.NETCore.Platforms" Version="3.0.0-preview.19109.6">
```

Comment:
To confirm: None of the other toolsets have newer versions?

Reply:
They might, but the bot is working correctly for those. This one is currently busted due to some build issues in the corefx repo.